source/MRViewer/MRRenderLabelObject.cpp

```diff
@@ -227,18 +221,48 @@ void RenderLabelObject::renderLeaderLine_( const RenderParams& renderParams ) co
 
     const auto shift = objLabel_->getPivotShift();
     auto box = objLabel_->labelRepresentingMesh()->getBoundingBox();
-    applyPadding( box, cBackgroundPaddingPx * ( box.max.y - box.min.y ) / objLabel_->getFontHeight() );
+    applyPadding( box, objLabel_->getBackgroundPadding() * ( box.max.y - box.min.y ) / objLabel_->getFontHeight() );
     const std::vector<Vector3f> leaderLineVertices {
         { shift.x, shift.y, 0.f },
         { box.min.x, box.min.y, 0.f },
         { box.max.x, box.min.y, 0.f },
+        { box.min.x, box.max.y, 0.f },
+        { box.max.x, box.max.y, 0.f },
     };
     bindVertexAttribArray( shader, "position", llineVertPosBufferObjId_, leaderLineVertices, 3, dirtyLLine_ );
 
-    constexpr std::array<Vector2i, 2> llineEdgesIndices{
-        Vector2i{ 0, 1 },
-        Vector2i{ 1, 2 },
+    std::vector<Vector2i> llineEdgesIndices {
+        // underscore line
+        { 1, 2 },
     };
```

Comment:
mb make array as class field? and jsut control size of it here (it is always 3, but pass to GPU only number of needed)?

Reply:
Rewrote to use std::array.